tests/integration/cli.py

```diff
@@ -18,21 +18,18 @@ def test_add_list_delete_projects():
     projects = output.split(" ")
     assert PROJECT1 not in projects
     assert PROJECT2 not in projects
-    projects.remove('\n')  # remove \n from list
     size = len(projects)
 
     subprocess.check_call(['haas', 'project_create', PROJECT1])
     output = subprocess.check_output(['haas', 'list_projects'])
     projects = output.split(" ")
-    projects = map(str.strip, projects)  # strips off \n from project name
     assert PROJECT1 in projects
     assert PROJECT2 not in projects
     assert len(projects) == size + 1
 
     subprocess.check_call(['haas', 'project_delete', PROJECT1])
     output = subprocess.check_output(['haas', 'list_projects'])
     projects = output.split(" ")
-    projects.remove('\n')  # remove \n from list
```

Comment:
While interfacing the client library with CLI, I had changed the way calls output results. 
because of that the said statement is no longer required. Infact the integration tests were failing when I made the suggested changes, because of the above reason. Just to make sure I confirmed with @naved001 before committing the changes.